src/components/notifications/AppNotifications/AppNotifications.scss

```diff
@@ -14,7 +14,7 @@
     gap: 12px;
     width: 100%;
     align-items: center;
-    padding: 8px;
+    padding: 10px 4px;
```

Comment:
Can we use relative values like `em` here?

Reply:
For sure 🫡